thirdeye-ui/src/app/components/anomaly-list-v1/anomaly-list-v1.component.tsx

```diff
@@ -92,12 +94,16 @@ export const AnomalyListV1: FunctionComponent<AnomalyListV1Props> = ({
 
     const anomalyNameRenderer = useCallback(
         (cellValue: Record<string, unknown>, data: UiAnomaly): ReactNode => {
-            const { isIgnored, id } = data;
+            const { isIgnored, isFlaggedAsNotAnAnomaly, id } = data;
 
             return addMutedStyle(
                 linkRendererV1(
                     `${t("label.entity-id", { id: cellValue })}${
                         isIgnored ? `(${t("label.ignored")})` : ""
+                    }${
+                        !isIgnored && isFlaggedAsNotAnAnomaly
+                            ? `(${t("label.flagged")})`
+                            : ""
                     }`,
```

Comment:
Let's make this more verbose such as:

```ts
let label = t("label.entity-id", { id: cellValue });

if (isIgnored) {
  label += `(${t("label.ignored")})`
}

if (!isIgnored && isFlaggedAsNotAnAnomaly) {
  label += `(${t("label.flagged")})`
}

return addMutedStyle(linkRendererV1(label), ....
```

Reply:
This looks easier to read. Updated accordingly.

Will try to keep things simpler like this for future PRs